workflow/rules/quantify.smk

```diff
@@ -64,7 +63,10 @@ rule finish_quantify:
     input:
         primir_intersect_sam=OUT_DIR / "{sample}" / "alignments_intersecting_primir.sam",
         mirna_intersect_sam=OUT_DIR / "{sample}" / "alignments_intersecting_mirna.sam",
-        table=OUT_DIR / "TABLES" / "all_{mir}_counts.tab",
+        table=expand(
+            OUT_DIR / "TABLES" / "all_{mir}_counts.tab",
+            mir=[mir for mir in config["mir_list"] if mir != "isomir"],
```

Comment:
But this looks like something unrelated to the environment issue? Are you trying to sneak something past me here? :raised_eyebrow::laughing: 

Or is this some artifact from another PR?